docker_entrypoint.sh

```diff
@@ -119,6 +156,8 @@ if [ -e "$FILE" ] ; then {
 } 
 fi
 
+exec busybox crond -f -l 0 -L /dev/stdout
```

Comment:
I would guess this `exec` breaks sigterm handling....

Reply:
Fixed